UPGRADE-2.0.md

```diff
@@ -13,6 +11,16 @@
 - The `project` option has been removed.
 - The `extra_data` option has been removed in favour of setting additional data
   directly in the context.
+- The `curl_method` option has been removed in favour of leaving to the user the
+  choice of setting an HTTP client supporting syncronous, asyncronous or both
+  transport methods.
+- The `curl_path` option has been removed.
+- The `curl_ipv4` option has been removed.
+- The `curl_ssl_version` option has been removed.
+- The `verify_ssl` option has been removed.
```

Comment:
Do we have any generic replacement for the `verify_ssl` option? Or do it depends now on the concrete client that is used?